### PR TITLE
feat: add `hexo` completion spec

### PR DESCRIPTION
**Pull request by patrick330602**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

feature: autocomplete for `hexo-cli` for [Hexo](https://hexo.io)

**What is the current behavior? (You can also link to an open issue here)**

does not provide autocomplete for hexo-cli

**What is the new behavior (if this is a feature change)?**

provides autocomplete for hexo-cli commands

### CODE REVIEW

File: src/hexo.ts [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+  script: "hexo list post | grep -E ^Draft",
+  postProcess: (out) => {
+    return out.split("\n").map(function (file) {
+      const title = file
+        .replace(/^.*   *_drafts\//g, "")
+        .replace(/  *(.*)$/g, "")
+        .replace(/.md$/g, "");
+      return {
+        name: title,
+        description: "draft for " + title,
```
Comment (SeparateRecords) [inline src/hexo.ts:11]: Just a quick case change 🙂 (we try to keep to "conventional descriptions", eg. upper case first letters, no periods at the end)
```suggestion
        description: "Draft for " + title,
```

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
Amazing!! Just one quick change and then I think it's ready to merge!

File: src/hexo.ts (hunk 1 above)
Comment (patrick330602) [reply src/hexo.ts]: Gotcha, and updated the description

File: src/hexo.ts [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    },
+    {
+      name: "--cwd",
+      description: "Customizes the path of current working directory",
+      args: {
+        name: "directory",
+        template: "folders",
+      },
+    },
+  ],
```
Comment (SeparateRecords) [inline src/hexo.ts:308]: One more thing, these look like they're global options - could you add `isPersistent` to them? Sorry for that 😅

Comment (patrick330602) [reply src/hexo.ts]: Gotcha and added

**Review by SeparateRecords — APPROVED** [review]

**Comment by SeparateRecords** [comment]
Thanks so much for adding this! Looks good 👍

**Pull request merged**